go/vt/vtorc/config/config.go

```diff
@@ -121,6 +121,24 @@ var (
 		},
 	)
 
+	backendReadConcurrency = viperutil.Configure(
+		"backend-read-concurrency",
+		viperutil.Options[int64]{
+			FlagName: "backend-read-concurrency",
+			Default:  32,
+			Dynamic:  true,
```

Comment:
Rest is all good, but I don't think this field should be dynamic. Since we initialize the semaphore with value we read for this config once, even if the users change it later, the change won't be affected.

Reply:
@GuptaManan100 good point, updated 👍